seed/ReservationSeeder.ts

```diff
@@ -0,0 +1,41 @@
+// import mongoose from 'mongoose';
```

Comment:
if it is not needed, please remove this file, or commented lines

Reply:
OK i will remove this script and will push with updated on next branch